man/man7/fi_provider.7

```diff
@@ -266,6 +270,6 @@ Logged if configured with the \[en]enable\-debug flag.
 .PP
 \f[C]fi_gni\f[R](7), \f[C]fi_hook\f[R](7), \f[C]fi_psm\f[R](7),
 \f[C]fi_sockets\f[R](7), \f[C]fi_usnic\f[R](7), \f[C]fi_verbs\f[R](7),
-\f[C]fi_bgq\f[R](7),
+\f[C]fi_bgq\f[R](7), \f[C]fi_opx\f[R](7)
```

Comment:
Likewise, the contents of this file will be replaced by post-processing on the .md file.